vcd/resource_vcd_nsxt_network_imported_test.go

```diff
@@ -16,7 +16,7 @@ func TestAccVcdNsxtNetworkImported(t *testing.T) {
 		t.Skip(t.Name() + " requires system admin privileges")
 	}
 
-	skipNoNsxtConfiguration(t)
+	//skipNoConfiguration(t)
```

Comment:
Just to be sure this is not a leftover comment or something

Reply:
removed